components/google_sheets/sources/new-row-added-shared-drive/new-row-added-shared-drive.mjs

```diff
@@ -0,0 +1,25 @@
+import httpBase from "../common/http-based/drive.mjs";
+import newRowAdded from "../common/new-row-added.mjs";
+
+export default {
+  ...httpBase,
+  ...newRowAdded,
+  key: "google_sheets-new-row-added-shared-drive",
+  name: "New Row Added (Shared Drive, Instant)",
+  description: "Emit new events each time a row or rows are added to the bottom of a spreadsheet in a shared drive",
+  version: "0.0.1",
+  dedupe: "unique",
+  type: "source",
+  hooks: {
+    ...httpBase.hooks,
+    ...newRowAdded.hooks,
+  },
```

Comment:
Since `newRowAdded` doesn't have any hooks, and the hooks from `httpBase` will be included anyway, this hooks section can be eliminated.
```suggestion
```